web/vibe/web/rest.d

```diff
@@ -711,7 +755,7 @@ class RestInterfaceClient(I) : I
 		 * Returns:
 		 *     The Json object returned by the request
 		 */
-		Json request(HTTPMethod verb, string name,
+		string request(HTTPMethod verb, string name,
```

Comment:
I didn't catch that this also is a breaking change due to the `protected` atttribute. I'm also not sure why it is `protected` in the first place, though, so we might just deprecate it and add the new signature as a private method (and go for a string output range instead of a `string` return value to prepare for dropping `appender` in favor of a `StreamOutputRange` or similar).

The old deprecated method would then only be called if no custom serialization is in effect.

Reply:
okay, just let quickly ask, if I understood correctly:
1. we lwill eave the original protected 'request' method in place, and we will implement it by using a call into our new private 'request' method
2. the new private 'request' method will no longer return a string, but it will accept a new parameter of type StreamOutputRange

is this correct?

Reply:
For number two I was thinking of a generic output range argument, but other than that that sounds correct.

The only issue is that the `protected request()` will have to re-parse the result to `Json` if it just calls the private one. So maybe we can move the serialization independent code to a separate method that is called by both and then let the old request just do its serialization the old way, instead of calling the new `private` one.

Reply:
I was just thinking and I think we should pass an InputRange(or similar) to the deserializer. So shouldn't we should return HTTPClientResponse from the private 'request' method? and later we can directly pass the InputStream from the HTTPClientResponse to the deserializer.

Reply:
although I am not sure if InterfaceProxy!InputStream can be converted to InputRange